scripts/obtain/dotnet-install.ps1

```diff
@@ -50,7 +50,7 @@
 #>
 [cmdletbinding()]
 param(
-   [string]$Channel="preview",
+   [string]$Channel="rel-1.0.0",
```

Comment:
If you are pulling preview2 bits I recommend you use the install script from the preview2 channel to keep things working there